notebooks/automatic_embedding_tei_inference_endpoints.ipynb

```diff
@@ -0,0 +1,845 @@
+{
```

Comment:
<p>Let's say "Hugging Face offers a number of GPUs that you can choose from" instead of "We have"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>